drivers/unix/ip_unix.cpp

```diff
@@ -38,19 +38,7 @@
   #include <windows.h>
   #include <stdio.h>
   #ifndef UWP_ENABLED
-    #if defined(__MINGW32__ ) && (!defined(__MINGW64_VERSION_MAJOR) || __MINGW64_VERSION_MAJOR < 4)
-      // MinGW-w64 on Ubuntu 12.04 (our Travis build env) has bugs in this code where
```

Comment:
Yeah, well, I actually did, I just hoped that you did that out of frustration :wink: .
I'll fix this soon :+1: